crates/sui-bridge-cli/src/main.rs

```diff
@@ -187,6 +195,77 @@ async fn main() -> anyhow::Result<()> {
             return Ok(());
         }
 
+        BridgeCommand::PrintBridgeRegistrationInfo { sui_rpc_url } => {
+            let sui_bridge_client = SuiClient::<SuiSdkClient>::new(&sui_rpc_url).await?;
+            let bridge_summary = sui_bridge_client
+                .get_bridge_summary()
+                .await
+                .map_err(|e| anyhow::anyhow!("Failed to get bridge summary: {:?}", e))?;
+            let move_type_bridge_committee = bridge_summary.committee;
+            let sui_client = SuiClientBuilder::default().build(sui_rpc_url).await?;
+            let stakes = sui_client
+                .governance_api()
+                .get_committee_info(None)
+                .await?
+                .validators
+                .into_iter()
+                .collect::<HashMap<_, _>>();
+            let names = sui_client
+                .governance_api()
+                .get_latest_sui_system_state()
+                .await?
+                .active_validators
+                .into_iter()
+                .map(|summary| {
+                    let protocol_key =
+                        AuthorityPublicKeyBytes::from_bytes(&summary.protocol_pubkey_bytes)
+                            .unwrap();
+                    (summary.sui_address, (protocol_key, summary.name))
+                })
+                .collect::<HashMap<_, _>>();
+            let mut authorities = vec![];
+            for (_, member) in move_type_bridge_committee.member_registration {
```

Comment:
you could work off references if you wanted particularly considering most of the usage is by reference. 
Just commenting to be honest, I do not think it matters much given the usage

Reply:
yeah we consume it anyway here